static/redfish/v1/schema/OemComputerSystem_v1.xml

```diff
@@ -26,46 +26,91 @@
                 <Property Name="IBM" Type="OemComputerSystem.IBM"/>
             </ComplexType>
 
+            <ComplexType Name="OpenBmc" BaseType="Resource.OemObject">
+                <Annotation Term="OData.AdditionalProperties" Bool="true" />
+                <Annotation Term="OData.Description" String="Oem properties for OpenBmc." />
+                <Annotation Term="Redfish.DynamicPropertyPatterns">
+                    <Collection>
+                        <NavigationProperty Type="OemComputerSystem.FirmwareProvisioning">
+                            <Annotation Term="OData.AutoExpand"/>
+                            <PropertyValue Property="Pattern" String="[A-Za-z0-9_.:]+" />
+                            <PropertyValue Property="Type" String="OemComputerSystem.FirmwareProvisioning" />
+                        </NavigationProperty>
+                    </Collection>
+                </Annotation>
+            </ComplexType>
+
+            <ComplexType Name="FirmwareProvisioning" BaseType="Resource.OemObject">
```

Comment:
What about upsteam here?

Reply:
This part `#OemComputerSystem.OpenBmc` and its descendants (e.g. `OemComputerSystem.FirmwareProvisioning`) haven't changed from upstream, and they are good.